Composer/packages/server/src/locales/en-US.json

```diff
@@ -125,8 +125,8 @@
   "add_additional_url_bdfac25d": {
     "message": "Add additional URL"
   },
-  "add_alternative_phrasing_47234f77": {
-    "message": "add alternative phrasing"
+  "add_alternative_phrasing_17e0304c": {
```

Comment:
should this file be changed?

Reply:
It looks like we have another l10n bubble. I'll get a new PR out to fix that first.